RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleRelPFIsoScaledCut.cc

```diff
@@ -0,0 +1,78 @@
+#include "PhysicsTools/SelectorUtils/interface/CutApplicatorWithEventContentBase.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "RecoEgamma/EgammaTools/interface/EffectiveAreas.h"
+
+
+class GsfEleRelPFIsoScaledCut : public CutApplicatorWithEventContentBase {
+  public:
+    GsfEleRelPFIsoScaledCut(const edm::ParameterSet& c);
+    
+    result_type operator()(const reco::GsfElectronPtr&) const final;
+
+    void setConsumes(edm::ConsumesCollector&) final;
+    void getEventContent(const edm::EventBase&) final;
+
+    double value(const reco::CandidatePtr& cand) const final;
+
+    CandidateType candidateType() const final { 
+      return ELECTRON; 
+    }
+
+  private:
+    const float _barrelC0, _endcapC0, _barrelCpt, _endcapCpt, _barrelCutOff;
+    bool _isRelativeIso;
+    EffectiveAreas _effectiveAreas;
+    edm::Handle<double> _rhoHandle;
```

Comment:
please do not use leading underscore in variable names (trailing is good)
https://cms-cpt-software.web.cern.ch/cms-cpt-software/General/CodingAndStyleRules.pdf

Reply:
done

Reply:
(note that in my original commit I had the leading underscores because I based my implementation on the already existing GsfEleEffAreaPFIsoCut.cc in the same directory, which also had leading underscores)